packages/checkbox/theme/lumo/vaadin-checkbox-styles.js

```diff
@@ -156,6 +156,12 @@ registerStyles(
       color: var(--lumo-contrast-30pct);
     }
 
+    :host([disabled]) [part='label'],
+    :host([disabled]) [part='helper-text'] {
+      color: var(--lumo-disabled-text-color);
```

Comment:
Now that this is explicitly defined for label and helper text, it could be removed from `:host([disabled]) {`

Reply:
Done.